DiscordIan/Ian.cs

```diff
@@ -108,6 +108,7 @@ private static ServiceProvider ConfigureServices(string[] args)
             // project services
             services.AddSingleton<CommandHandlingService>();
             services.AddTransient<FetchJsonService>();
+            services.AddTransient<FetchXMLService>();
```

Comment:
This should probably be FetchXmlService following the [Framework design guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions).